spec/index.bs

```diff
@@ -3675,7 +3675,7 @@ GPUComputePipeline includes GPUPipelineBase;
 
 <script type=idl>
 dictionary GPUComputePipelineDescriptor : GPUPipelineDescriptorBase {
-    required GPUProgrammableStageDescriptor computeStage;
+    required GPUProgrammableStage computeStage;
```

Comment:
nit: maybe rename to `compute` to match `fragment` and `vertex` in the `GPURenderPipelineDescriptor`?

Reply:
oops, nice catch!